lib/elastomer/client/warmer.rb

```diff
@@ -66,6 +72,7 @@ def get(params = {})
       # See https://github.com/elasticsearch/elasticsearch/issues/5155
       def exists?
         response = get
+
```

Comment:
Is this newline intentional?

Reply:
I can remove it. I originally had some code here before I realized warmers were Just Gone in 5.